src/workloads/execution/FilterWithComplexLogicalExpression.yml

```diff
@@ -517,6 +544,126 @@ Actors:
           OperationCommand:
             Filter: {"n1.a.b.c.d.z": 0}
 
+- Name: AggregationExpressionFiftyClauseRootedOr
+  Type: CrudActor
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [17]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Collection: *Collection
+        Operations:
+        - OperationName: find
+          OperationCommand:
+            LoadConfig:
+              Path: "FilterWithComplexLogicalExpression.yml"
```

Comment:
[question] The self-reference here is a bit wacky, though I do think there is some value in keeping the template in this file instead of moving it into its own file. Is there no way to omit this parameter if the template is defined in the same file? If not, should we introduce something like this?

Reply:
Nice! This is definitely an improvement. Resolving the thread.